src/dev/display/lcd/lv_lcd_panel_commands.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
As these are specific to st7789, please move these commands in `lv_st7789.c` or at least to the `st7789` folder

Reply:
These commands are common for various controllers, not ST7789 specific.

Reply:
But not all, so we can't put it into a generic LCD file. 
It seems roughly 10 commands are really used in st7789.c. I think it's okay to define them locally to avoid any conflicts later.